modules/Planner/resources_add_ajaxProcess.php

```diff
@@ -34,19 +34,19 @@
         exit();
     } else {
         //Proceed!
-        $id = $_POST['id'];
-        $type = $_POST[$id.'type'];
+        $id = $_POST['id'] ?? '';
+        $type = $_POST[$id.'type'] ?? '';
         if ($type == 'File') {
-            $content = $_FILES[$id.'file'];
+            $content = $_FILES[$id.'file'] ?? '';
         } elseif ($type == 'Link') {
-            $content = $_POST[$id.'link'];
+            $content = $_POST[$id.'link'] ?? '';
         }
-        $name = $_POST[$id.'name'];
-        $category = $_POST[$id.'category'];
-        $purpose = $_POST[$id.'purpose'];
-        $tags = strtolower($_POST[$id.'tags']);
-        $gibbonYearGroupIDList = (!empty($_POST[$id.'gibbonYearGroupID']))? implode(',', $_POST[$id.'gibbonYearGroupID']) : '';
-        $description = $_POST[$id.'description'];
+        $name = $_POST[$id.'name'] ?? '';
+        $category = $_POST[$id.'category'] ?? '';
+        $purpose = $_POST[$id.'purpose'] ?? '';
+        $tags = strtolower($_POST[$id.'tags'] ?? '');
+        $gibbonYearGroupIDList = implode(',', $_POST[$id.'gibbonYearGroupID']) ?? '';
```

Comment:
The null coalesce is still outside the function in this instance. It needs to be applied directly to the $_POST variable. Can you take a closer look and update it? Thanks